govstack_test_harness_api/building_blocks/bb-digital-registries/views/check_if_record_exists_in_registry.py

```diff
@@ -0,0 +1,33 @@
+from django.http import JsonResponse
+from rest_framework.decorators import api_view
+from ..serializers.record_exists_serializer import RecordExistsSerializer
+from drf_yasg.utils import swagger_auto_schema
+from .swaggers_schema import exists_request_body, exists_response_body
+from ..swaggers_schema import info_mediator_client
+from ..services.services import login_with_env_variables
+from ..controllers.record_exists_controller import check_if_record_exists
```

Comment:
Please replace relative imports with absolute ones.
If you're using PyCharm IDE it can be set as an default in settings.